src/embed/main.js

```diff
@@ -49,6 +49,14 @@ worldRenderer.hotspotRenderer.on('click', onHotspotClick);
 
 window.worldRenderer = worldRenderer;
 
+// Omnitone.
+// TODO(hoch): refactor these globals.
+var Omnitone = 
+  require('../../node_modules/omnitone/build/omnitone.min.js').Omnitone;
```

Comment:
Not really specific to this PR, more a question about Omnitone: does omnitone export something other than the Omnitone class?